cmd/gosky/main.go

```diff
@@ -79,6 +79,9 @@ func run(args []string) {
 			EnvVars: []string{"ATP_PLC_HOST"},
 		},
 	}
+
+	// TODO: slog.SetDefault from param/env
```

Comment:
way less important for `gosky` vs relay, IMO, though @whyrusleeping might care? might be helpful for some debug stuff

Reply:
I have a second PR coming with this, at least for relay